### PR TITLE
Utilizing nullable reference types

### PR DESCRIPTION
**Pull request by tmilnthorp**
Per #688

### CODE REVIEW

**Comment by tmilnthorp** [comment]
@angularsen I am not sure how to fix this build

**Comment by angularsen** [comment]
Well, the error says
> CSC : error CS8630: Invalid 'nullable' value: 'Enable' for C# 7.3. Please use language version 'preview' or greater. 

I'll try pushing a language version fix and see if it helps.

**Comment by angularsen** [comment]
It builds locally here, so must be environment specific.

**Comment by angularsen** [comment]
Weird, I just upgraded to VS2019 image and it.. worked. I tried earlier and it.. well, didn't work :-) Maybe the appveyor images have been improved since then.

**Comment by tmilnthorp** [comment]
Glad it works! Everything ok to merge? I just did the main UnitsNet project for now. We can work on projects in parts.

I'm also guessing that component runtime doesn't support this.

File: CodeGen/Generators/UnitsNetGen/StaticQuantityGenerator.cs [hunk 1]
```diff
@@ -22,6 +22,8 @@ public override string Generate()
 using UnitsNet.InternalHelpers;
 using UnitsNet.Units;
 
+#nullable enable
```
Comment (angularsen) [inline CodeGen/Generators/UnitsNetGen/StaticQuantityGenerator.cs:25]: Shouldn't we set this on the project instead?
https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/nullable-reference-types

Or are there so many issues to fix that we will have to introduced this in bits and pieces?

Comment (angularsen) [reply CodeGen/Generators/UnitsNetGen/StaticQuantityGenerator.cs:25]: Other than that, this looks pretty good to me and ready to merge.

Comment (tmilnthorp) [reply CodeGen/Generators/UnitsNetGen/StaticQuantityGenerator.cs:25]: It is set on the project. However without this in the generated files, I receive a compiler error:

```
Error CS8669 The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source.
```

I'm guessing the .g. annotation tells the compiler it's generated?

Comment (angularsen) [reply CodeGen/Generators/UnitsNetGen/StaticQuantityGenerator.cs:25]: Aha, yes it probably does. Today I learned.

**Comment by angularsen** [comment]
[Release UnitsNet/4.62.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.62.0)

**Pull request merged**